speech/google/cloud/speech/_gax.py

```diff
@@ -106,21 +104,27 @@ def async_recognize(self, sample, language_code=None,
         :returns: Instance of ``Operation`` to poll for results.
         """
         config = RecognitionConfig(
-            encoding=sample.encoding, sample_rate=sample.sample_rate,
-            language_code=language_code, max_alternatives=max_alternatives,
+            encoding=sample.encoding,
+            language_code=language_code,
+            max_alternatives=max_alternatives,
             profanity_filter=profanity_filter,
-            speech_context=SpeechContext(phrases=speech_context))
+            sample_rate_hertz=sample.sample_rate_hertz,
+            speech_contexts=[SpeechContext(phrases=speech_contexts)],
```

Comment:
I just checked with the team -- it is a nested list but the outer list can only contain one element.